README.md

```diff
@@ -35,13 +35,14 @@ These instructions assume Ubuntu Trusty 64.
 
 ```
 sudo apt-get install -y git-core
-sudo apt-get install -y libtiff5-dev libjpeg8-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python-tk # Requirements for pillow
 sudo apt-get install -y python3-dev
 
 git clone https://github.com/OpenDataServices/grantnav.git
 cd grantnav
-python3 -m venv .ve --python=/usr/bin/python3
+python3 -m venv .ve
 source .ve/bin/activate
+# Make sure you have a recent version of pip, to install binary wheel packages.
```

Comment:
@Bjwebb it'd be helpful to know what's considered 'recent' - certainly in CentOS world, just typing 'yum update python-pip' doesn't get you what the Python community would consider to be recent.

Reply:
Oh, wait, ignore me.